src/views/privacypolicy/privacypolicy.jsx

```diff
@@ -203,6 +203,7 @@ var Privacypolicy = React.createClass({
                              Policy on a regular basis.
                         </p>
                     </section>
+                    <p><b>The Scratch Privacy Policy was last updated: October 2016</b></p>
```

Comment:
@TheGrits my bad - I thought the text on the Terms of Use was bolded :( Can you unbold this text to make it consistent with that page? Sorry for the back and forth.